common/corpus/types.txt

```diff
@@ -1129,7 +1129,7 @@ type A<B, C> = B extends C
     (template_literal_type
       (template_type
         (literal_type
-          (string)))
+          (string (string_fragment))))
```

Comment:
Tests were not passing locally (`npm run build-typescript && npm run build-tsx`) because of this `(string_fragment)` even though I had changed this file

Reply:
I managed to fix it but not sure if it's proper. Probably would benefit from a second review.